src/locales/translations.json

```diff
@@ -858,6 +933,11 @@
         "pt-br": "Exemplo: Dúvidas",
         "en": "Example: Doubts",
         "es": "Ejemplo: Dudas"
+      },
+      "tag": {
+        "pt-br": "Adicionar",
+        "en": "Add",
+        "es": "Agregar"
```

Comment:
This traduction is already available as `add` in this file, so there is no need to repeat it here.